contracts/core/liquidators/LinearAuctionLiquidator.sol

```diff
@@ -87,15 +87,19 @@ contract LinearAuctionLiquidator is LinearAuction, ILiquidator {
      * @param _currentSet                   The Set to rebalance from
      * @param _nextSet                      The Set to rebalance to
      * @param _startingCurrentSetQuantity   The currentSet quantity to rebalance
+     * @param _liquidatorData                  Bytecode formatted data with liquidator-specific arguments
      */
     function startRebalance(
         ISetToken _currentSet,
         ISetToken _nextSet,
-        uint256 _startingCurrentSetQuantity
+        uint256 _startingCurrentSetQuantity,
+        bytes calldata _liquidatorData
     )
         external
         isValidSet
     {
+        _liquidatorData; // Pass linting
```

Comment:
maybe we create a `handleLiquidatorData` function that this passes to? And then it doesn't nothing with it there. This feels kinda weird, having it in it's own function makes it feel more explicit about what's going on. Not super opinionated though.